package.json

```diff
@@ -76,12 +76,13 @@
         "babel-preset-stage-2": "^6.24.1",
         "babel-preset-vue": "^2.0.2",
         "cross-env": "^5.1.6",
-        "eslint": "^4.19.1",
+        "eslint": "^5.16.0",
+        "eslint-config-airbnb-base": "^13.1.0",
         "eslint-friendly-formatter": "^4.0.1",
```

Comment:
Strong opinions on this? Any objections to switching to `codeframe` for example?

Reply:
https://github.com/Ultimaker/Meta/blob/master/languages/javascript_code_conventions.md 🌵